src/utils.ts

```diff
@@ -104,12 +104,42 @@ export async function findNode(cwd: string): Promise<string> {
   // Stage 3: If we still haven't found Node.js, try to find it via a subprocess.
   // This evaluates shell rc/profile files and makes nvm work.
   node ??= await findNodeViaShell(cwd)
+
+  // If volta isn't installed in a Volta folder, this test will fail.
+  // If anyone got a better idea for checking volta's presence, please let me know.
+  const voltaRegex = /Volta/i
```

Comment:
I would check that it ends with Volta/node.EXE since it's hardcoded by Volta already

Reply:
I have updated to Volta\node.exe. Windows use backslash for path even in node. The test would fail otherwise.